boards/arm/nucleo_l433rc/support/openocd.cfg

```diff
@@ -0,0 +1,7 @@
+source [find interface/stlink.cfg]
+
+transport select hla_swd
+
+source [find target/stm32l4x.cfg]
+
+reset_config srst_only
```

Comment:
does debug work well? Wondering if entries such as https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nucleo_f103rb/support/openocd.cfg#L3 are needed.

Reply:
I followed the template of other nucleo boards in the repo, like `nucleo_l432kc`